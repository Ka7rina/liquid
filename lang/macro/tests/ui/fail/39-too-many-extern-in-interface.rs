use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        fn bar();
    }

    extern "liquid" {
        fn baz();
    }
}

fn main() {}
