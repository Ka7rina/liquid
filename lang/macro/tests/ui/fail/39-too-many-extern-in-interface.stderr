error: the number of `extern` blocks in interface module should not exceed 1
  --> $DIR/39-too-many-extern-in-interface.rs:9:5
   |
9  | /     extern "liquid" {
10 | |         fn baz();
11 | |     }
   | |_____^
